src/main/webapp/app/shared/util/role.util.ts

```diff
@@ -0,0 +1,35 @@
+export class RoleUtil {
```

Comment:
is it necessary to have a wrapping class? couldn't we just export all these as constants? (I don't know if its possible in ts, but see `roles.constants.ts` in `constants`)

Reply:
yup hast recht.